### PR TITLE
js to draw bounds for nets

### PR DESCRIPTION
**Pull request by antonleviathan**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -62,11 +62,35 @@ function drawNet(container, states, transitions, marking) {
...
+    boundary[3] = Math.max(boundary[3], y)
+  }
+
+  _.forEach(states, function(el){
+    updateBoundary(el.x, el.y)
+  })
+  _.forEach(transitions, function(el){
+    updateBoundary(el.x, el.y)
+  })
+
```
Comment (wires) [inline js/app.js]: should add a padding, so
```js
function pad(boundary, padding) {
  return [
    boundary[0] - pad, boundary[1] + pad,
    boundary[2] - pad, boundary[3] + pad
  ]
}

boundary = pad(boundary, r)
```

**Review by wires — COMMENTED** [review]
at the top of `app.js` there is `drawNet(..)`, rename to `drawNet_(states, transitions, ..., width, height)` and make sure you can pass width, height.

then outside of this function, define `drawNet(..)` that computes the boundary and then translates the `states` and `transitions` coordinates. it then calls `drawNet_(translatedStates_, translatedTransitions_, ..., computedWidth, computedHeight)`

File: js/app.js (hunk 1 above)
Comment (antonleviathan) [reply js/app.js]: Done!

**Comment by antonleviathan** [comment]
@wires This is why it's broken haha we haven't merged this and the other branch. Gotta fix the links though.

**Pull request merged**